includes/class-amp-travel-taxonomies.php

```diff
@@ -146,7 +145,7 @@ public function save_activity_meta( $term_id ) {
 	 * Register 'activity' and 'location' taxonomy.
 	 */
 	public function register_taxonomies() {
-		register_taxonomy( 'activity', array( 'adventure', 'post' ), array(
+		register_taxonomy( 'activity', array( AMP_TRAVEL_CPT::POST_TYPE_SLUG_SINGLE, 'post' ), array(
```

Comment:
Thanks for being proactive and changing this to `AMP_TRAVEL_CPT::POST_TYPE_SLUG_SINGLE`.